src/renderer/shared/ui/Popovers/Popover/Popover.tsx

```diff
@@ -26,7 +30,28 @@ export const Popover = ({
 
   // prevents modal flickering when just passing across popup button
   // and gives user more time to move cursor to Popup.Panel
-  const debouncedIsOpen = useDebounce(isOpen, 200);
+  const debouncedIsOpen = useDebounce(isOpen, 100);
+  const parentRect = ref.current?.getBoundingClientRect();
+  const horizontalAlign = horizontalPos !== 'right' && {
+    transform: `translateX(${horizontalPos === 'center' ? '-50%' : '-100%'})`,
+  };
+
+  useEffect(() => {
```

Comment:
Maybe create react hook with this effect? Looks like it can be useful if we will do same things in other components